src/Elastic.Elasticsearch.Managed/ConsoleWriters/LineOutParser.cs

```diff
@@ -64,7 +64,7 @@ public static class LineOutParser
 		private static bool TryGetStartedConfirmation(string section, string message)
 		{
 			var inNodeSection = section == "o.e.n.Node" || section == "node";
-			return inNodeSection && message == "started";
+			return inNodeSection && (message == "started" || message.StartsWith("started {", StringComparison.Ordinal));
```

Comment:
https://github.com/elastic/elasticsearch/pull/85773/files#diff-d01b960cab84fe2338f230b5e7535bc8efa9f9778ecf61c9f527067351dca895R1270 the `{}` is a placeholder and will be replaced by the node name. 
```suggestion
			return inNodeSection && (message == "started" || message.StartsWith("started ", StringComparison.Ordinal));
```

Reply:
The final log ends up with something like this:
```
[2022-04-12T11:04:38,180][INFO ][o.e.n.Node               ] [readonly-node-e98b5e9200] started {readonly-node-e98b5e9200}{5mg-DiUAQhGbXUVC5OIm2A}{Y5ID3JtKSluPGolYUL5QDQ}{127.0.0.1}{127.0.0.1:9300}{cdfhilmrstw}{gateway=true, xpack.installed=true, ml.max_jvm_size=1073741824, testingcluster=true, ml.machine_memory=17115570176}
```

So I believe checking for the open brace should be okay?